### PR TITLE
Have nscssproperty a method of data instances

### PR DESCRIPTION
**Pull request by upsuper**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/20619)

### CODE REVIEW

File: components/style/properties/properties.mako.rs [hunk 1]
```diff
@@ -1578,10 +1572,10 @@ impl PropertyId {
                 % for (kind, properties) in [("Longhand", data.longhands), ("Shorthand", data.shorthands)]:
                     % for property in properties:
                         "${property.name}" => StaticId::${kind}(${kind}Id::${property.camel_case}),
-                        % for name in property.alias:
-                            "${name}" => {
+                        % for alias in property.alias:
```
Comment (emilio) [inline components/style/properties/properties.mako.rs]: Wait, the prefixed code does:

```
 property.alias.append('-%s-%s' % (prefix, property.name))
```

So basically there's no object there, how does that work?

Comment (upsuper) [reply components/style/properties/properties.mako.rs]: That happens before we convert the strings into objects.

**Comment by upsuper** [comment]
Looks like heycam probably isn't going to get to this today.

r? @emilio

File: components/style/properties/properties.mako.rs (hunk 1 above)
Comment (emilio) [reply components/style/properties/properties.mako.rs]: Oh, alright, makes sense.

Thank you, this is a nice cleanup :)

**Comment by emilio** [comment]
(And sorry for the lag, just saw this)

**Pull request merged**